crawl-ref/source/godabil.cc

```diff
@@ -4242,48 +4244,85 @@ bool gozag_setup_potion_petition(bool quiet)
     return true;
 }
 
-bool gozag_potion_petition()
+// What's the price of a particular potion effect?
+int _gozag_price_effect(item_def potion)
```

Comment:
godabil.cc: In function ‘int _gozag_price_effect(item_def)’:
    godabil.cc:4248:5: warning: no previous declaration for ‘int _gozag_price_effect(item_def)’ [-Wmissing-declarations]
 int _gozag_price_effect(item_def potion)

Not sure how to fix this